### PR TITLE
dealt with #89 #101

### LINKED ISSUE #89: Add to destroy to act types
rework entry act 45

### PR DESCRIPTION
**Pull request by chiayiwu**
close #89

### CODE REVIEW

**Comment by inamoto4** [comment]
Thank you for completing the male/female column, this helps! 

Why did you replace commas through semicolons though? 
Is it because of #85? I guess Duncan was pointing at sth different.
Take the following sentence as an example:
"after reading lots of foreign and Chinese classics; he still found Lu Xun the best"

I would say that we definitely need a comma here, not a semicolon.

File: data/Act.csv [hunk 1]
```diff
@@ -1,79 +1,79 @@
...
+A00060,P0153,AT01,T00077,,1966,1976,he wrote 'research of the Dream of the Red Chamber' and was put on the hat of 牛鬼蛇神. In 1954 a campaign was launched to criticise his 'bourgeoise' interpretation of the book.,,T00044,372,,2017-09-22,MG,2017-09-26,MG,,,,
+A00061,P0029,AT01,T00077,,,,He read it before his university studies and mentions that this book was one of the most-beloved and widely-read among his book friends.,,T00029,116-119,,2017-09-22,MG,2017-09-22,MG,,,,
+A00062,P0154,AT01,T00077,,,,He wanted the book so bad and it was really hard to have one. But he finally got one from the brother of one of his classmate and read it secretly.,,T00044,390,,2017-09-22,MG,2017-09-22,MG,,,,
+A00063,P0155,AT01,T00077,,1964,1965?,,,T00045,46,,2017-09-22,MG,2017-09-22,MG,,,,
+A00064,P0156,AT01,T00078,,1966,,after reading lots of foreign and Chinese classics, he still found Lu Xun the best,,T00027,ONLINE SOURCE,,2017-09-24,CW,2017-10-02,DP,,,
```
Comment (duncdrum) [inline data/Act.csv]: You columns are out of sync, avoid commas when inputting text

File: data/Act.csv [hunk 2]
```diff
@@ -1,79 +1,79 @@
...
+A00019,P0120,AT01,P0001,,1954,1956,for the next two years, he read much Lu Xun, esp. Essays,,T00030,57,,2017-07-28,LH,2017-07-28,LH,,
+A00020,P0120,AT10,P0001,,1967,1967,in 1967 he compiled 鲁迅语录 not for publications but for his friends,,T00030,21,,2017-07-28,LH,2017-07-28,LH,,,,
+A00021,P0106,LENA,P0001,,1967,1967,Later the author's home was plundered as well, all of his collections were confiscated by the red guards.,,T00005,569,,2017-07-28,LH,2017-10-02,DP,,,
+A00022,P0133,AT01,T00042,,1979,1979,The author read the peom of Baudelaire （Le Balcon 露台／阳台） at this time when so many foreign poems appeared again in public.,,T00044,533,,2017-08-07,MG,2017-10-02,DP,,,,
+A00023,P0133,AT01,T00041,,1984,1984,In a morning of the summer in 1984, the author visited Chen Jingrong, and she showed him a collection of her translation in the 1960s, and these Poems were once published on 'World Literature', which had influenced Bei Dao and other poets at that time.,,T00044,539,,2017-08-08,MG,2017-08-08,MG
```
Comment (duncdrum) [inline data/Act.csv]: s.b.

File: data/Act.csv [hunk 3]
```diff
@@ -1,79 +1,79 @@
...
+A00002,P0106,AT01,T00004,,1972,1976,mentions it among others as an important reading for him and his zhiqing fellows,,T00007,581,,2017-07-12,OS,2017-07-12,OS,,,,
+A00003,P0017,AT01,T00008,,1968,1978,mentions it among others as an important reading for the whole development in the field of literature after 1978,,T00009,219,,2017-07-12,OS,2017-07-12,OS,,,,
+A00004,P0108,AT01,T00010,,1972,1975,Xu Xiao read Ticket to the Stars and was very much influenced by it,,T00012,381,,2017-07-12,OS,2017-07-12,OS,,,,
+A00005,P0014,AT05,T00010,P0108,1972,1975,Zhao Yifan copied Ticket to the Stars and handed it to Xu Xiao,,T00012,381,,2017-07-12,OS,2017-07-12,OS,,,,
+A00006,P0109,AT04,T00013,,1969,1969,Shu Ting wants to go to Gorky‘s 'Universities' during her rustication,,T00014,300,,2017-07-19,LH,2017-10-02,DP,,,,
+A00007,P0020,AT04,T00013,,1966,1976,Gao Hua mentions that during the turmoil years of CR, when as a son of 'heiwulei' (five black categories) he has been living in the bottom of the society, receiving supercilious looks and being discriminated against, these two books by Gorky acted as his spiritual guides.,,T00006,133,,2017-07-19,LH,2017-10-02,DP,
```
Comment (duncdrum) [inline data/Act.csv]: s.b.

**Comment by duncdrum** [comment]
@inamoto4 no, I meant "no commas". The cells are no longer properly alligned in this PR (hit view and scroll right) The safest way to avoid this is not using commas, dots and semicolons are fine.

File: data/Act.csv (hunk 1 above)
Comment (inamoto4) [reply data/Act.csv]: Yes, I have seen this too, but when I open my Github Desktop now, the columns are in sync. Feels a bit irritating.

**Comment by inamoto4** [comment]
No commas in our whole database, you mean? As they are causing columns to get out of sync? 
Yet semicolons and commas denote sth different.

**Comment by duncdrum** [comment]
yes commas denote ``here be a cell``. You are free to rephrase comment so that they don't require a comma. Just make individual short sentences, it's better english style to boot. Afterall, its a database, not a multi-page essay.

File: data/Act.csv (hunk 1 above)
Comment (duncdrum) [reply data/Act.csv]: It depends on how you display the files in this branch as tables. If it doesn't display properly on Github, it won't do so in 100 other contexts. Point remains ``commas = bad``

**Comment by duncdrum** [comment]
Short: This still needs work.
Long: I already fixed the commas once, just to have them reverted. This is why I am asking people to prepare their own fixes.

**Comment by chiayiwu** [comment]
what's now left to be fixed to close this pull request? My GitHub is on strike again
image

**Comment by duncdrum** [comment]
@chiayiwu as the error message says you have changes in your files that need to be committed before switching branches

**Comment by chiayiwu** [comment]
Yes I will meet Mira this weekend, after I wrote the email we contacted
directly via whatsapp!


Mit freundlichen Grüßen / Best Regards,

Chia-yi Wu

2017-11-06 9:50 GMT+01:00 Duncan Paterson <notifications@github.com>:

**Pull request merged**